src/commonComponents/BrandLogo.js

```diff
@@ -0,0 +1,11 @@
+import "./BrandLogo.css";
+import brandLogo from ".././assets/images/swiggy-logo.svg";
+
+const brandName = "Foodistic";
```

Comment:
it could be present in constants

Reply:
I've kept it intentionally in this component so even if someone uses it for other applications they just have to mark the changes in this file. I'll rethink about it.